src/aom/templates/headers.js

```diff
@@ -0,0 +1,122 @@
+
+const name = "aom/templates/headers";
+
+const localizationStrings = {
+  en: {
+    author: "Author:",
+    authors: "Authors:",
+    editor: "Editor:",
+    editors: "Editors:",
+    former_editor: "Former editor:",
+    former_editors: "Former editors:",
+    latest_editors_draft: "Latest editor's draft:",
+    latest_published_version: "Latest approved version:",
+    this_version: "This version:",
+    issue_tracker: "Issue Tracker:",
+  },
+};
+
+export const l10n = getIntlData(localizationStrings);
+
+function getSpecSubTitleElem(conf) {
+  let specSubTitleElem = document.querySelector("h2#subtitle");
+
+  if (specSubTitleElem && specSubTitleElem.parentElement) {
+    specSubTitleElem.remove();
+    conf.subtitle = specSubTitleElem.textContent.trim();
+  } else if (conf.subtitle) {
+    specSubTitleElem = document.createElement("h2");
+    specSubTitleElem.textContent = conf.subtitle;
+    specSubTitleElem.id = "subtitle";
+  }
+  if (specSubTitleElem) {
+    specSubTitleElem.classList.add("subtitle");
+  }
+  return specSubTitleElem;
+}
+
+export default conf => {
+  return html`<div class="head">
+    ${conf.specStatus !== "PD" ? conf.logos.map(showLogo) : ""}
+    ${document.querySelector("h1#title")} ${getSpecSubTitleElem(conf)}
+    <h2>
+      ${conf.textStatus} -
+      <time class="dt-published" datetime="${conf.dashDate}"
+        >${conf.publishHumanDate}</time
+      >
+    </h2>
+    <dl>
+      <dt>${l10n.this_version}</dt>
+      <dd>
+        <a class="u-url" href="${conf.thisVersion}">${conf.thisVersion}</a>
+      </dd>
+      <dt>${l10n.issue_tracker}</dt>
+      <dd>
+        <a class="u-url" href="${conf.issueTracker}">${conf.issueTracker}</a>
+      </dd>
+      <dt>${conf.multipleEditors ? l10n.editors : l10n.editor}</dt>
+      ${showPeople(conf.editors)}
+      ${Array.isArray(conf.formerEditors) && conf.formerEditors.length > 0
+        ? html`
+            <dt>
+              ${conf.multipleFormerEditors
+                ? l10n.former_editors
+                : l10n.former_editor}
+            </dt>
+            ${showPeople(conf.formerEditors)}
+          `
+        : ""}
+      ${conf.authors
+        ? html`
+            <dt>${conf.multipleAuthors ? l10n.authors : l10n.author}</dt>
+            ${showPeople(conf.authors)}
+          `
+        : ""}
+      ${conf.otherLinks ? conf.otherLinks.map(showLink) : ""}
+    </dl>
+    ${renderCopyright(conf)}
+    <hr />
+  </div>`;
+};
+
+function renderCopyright(conf) {
+  // If there is already a copyright, let's relocate it.
+  const existingCopyright = document.querySelector(".copyright");
+  if (existingCopyright) {
+    existingCopyright.remove();
+    return existingCopyright;
+  }
+  if (conf.hasOwnProperty("overrideCopyright")) {
+    const msg = "The `overrideCopyright` configuration option is deprecated.";
+    const hint = 'Please use `<p class="copyright">` instead.';
+    showWarning(msg, name, { hint });
+  }
+  return conf.overrideCopyright
```

Comment:
I would recommend not supporting `.overrideCopyright` In the `conf` object. 

To override the copyright, one can do:
```
<p id="copyright">
  No copyleft intended  2020!
</p>
```

https://respec.org/docs/#copyrights-patents

Reply:
fixed.